modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/representative_users_controller.rb

```diff
@@ -4,7 +4,42 @@ module AccreditedRepresentativePortal
   module V0
     class RepresentativeUsersController < ApplicationController
       def show
-        render json: @current_user
+        # TODO: Move to serializer. Temporary quick and dirty.
+        render json: {
+          account: {
+            # Correct value?
+            account_uuid: @current_user.uuid
+          },
+          profile: {
+            first_name: @current_user.first_name,
+            last_name: @current_user.last_name,
+            verified:,
+            sign_in: {
+              service_name: @current_user.sign_in[:service_name]
+            }
+          },
+          # TODO: Should they get prefill for e.g. 21a?
+          prefills_available: [],
+          in_progress_forms:
+        }
+      end
+
+      private
+
+      def in_progress_forms
+        InProgressForm.submission_pending.where(user_uuid: @current_user.uuid).map do |form|
+          {
+            form: form.form_id,
+            metadata: form.metadata,
+            lastUpdated: form.updated_at.to_i
+          }
+        end
+      end
+
+      # Move to model? Logic taken from elsewhere.
```

Comment:
We should add a `verified` attribute to the `representative_user` and load it in the `representative_user_loader` instead of having the logic live in the controller.

You can add an `attribute :verified` to the [RepresentativeUser](https://github.com/department-of-veterans-affairs/vets-api/blob/08770b0986a1c3689d043ee150dd58c84e4cffb8/modules/accredited_representative_portal/app/models/accredited_representative_portal/representative_user.rb#L24-L25), then set it in the [RepresentativeUserLoader](https://github.com/department-of-veterans-affairs/vets-api/blob/c2b328d51b0ced210399f48f880a7543acd199fd/modules/accredited_representative_portal/app/services/accredited_representative_portal/representative_user_loader.rb#L63-L82).

You'd have to move the below logic to the `representative_user_loader`
```ruby
     def verified
        loa = @current_user.loa.to_h[:current]&.to_i
        loa == SignIn::Constants::Auth::LOA_THREE
      end
```

Let me know if you need help with that.